server/app.js

```diff
@@ -21,10 +21,22 @@ else {
 
 const app = express();
 const port = process.env.PORT | 3000;
+
+/* ADMINISTRATION CODE */
+function localhostHandler(request, response, next) {
+    response.header('Access-Control-Allow-Origin', '*');
+    response.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+    response.header('Access-Control-Allow-Methods', 'PUT, POST, GET, DELETE, OPTIONS');
+    next();
+}
+/*^^^^ ADMINISTRATION CODE ^^^^*/
+
 const routes = new Router().getRouter();
 function setupApp() {
     app.use(cors());
     app.use(bodyParser.json());
+    app.use(bodyParser.urlencoded({ extended: false }))     //use body parser to read url 
```

Comment:
This may cause possible unintended consequences. We need to test if it'll work on the main branch.